src/YoloDev.Expecto.TestSdk/YoloDev.Expecto.TestSdk.fsproj

```diff
@@ -20,9 +20,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Include="build/netcoreapp3.1/YoloDev.Expecto.TestSdk.props" Pack="true" PackagePath="build\netcoreapp3.1\" />
-    <None Include="build/netcoreapp3.1/_._" Pack="true" PackagePath="lib\netcoreapp3.1\" Visible="false" />
-    <None Include="build/net5.0/_._" Pack="true" PackagePath="lib\net5.0\" Visible="false" />
+    <None Include="build/net6.0/_._" Pack="true" PackagePath="lib\net6.0\" Visible="false" />
```

Comment:
Pretty sure this right here is your issue. The file `build/net6.0/_._` does not exist. The `net5.0` folder need to be renamed. And the `netcoreapp3.1` folder should probably be deleted.